Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "iota-crypto"
-version = "0.11.0"
+version = "0.12.0"
```

```suggestion
version = "0.11.0"
```
Do  not change the version manually, as long as a `.changes` is provided this will happen automatically.